gc/scheduler/scheduler.go

```diff
@@ -20,12 +20,17 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"strings"
 	"sync"
 	"time"
 
 	"github.com/containerd/containerd/gc"
 	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/plugin"
+	"github.com/containerd/containerd/tracing"
+
+	"go.opentelemetry.io/otel/attribute"
```

Comment:
There is a new SpanAttribute method defined in the tracing package which you can use. In that way we do not need to import `"go.opentelemetry.io/otel/attribute"` in all  places

https://github.com/containerd/containerd/blob/b2a01eedf4aa6f7a80aaa47a7096b180ee6c36e7/pull.go#L66-L70